Assets/Tilemap/Tiles/Rule Tile/Scripts/RuleTile.cs

```diff
@@ -86,7 +86,8 @@ public override bool StartUp(Vector3Int location, ITilemap tilemap, GameObject i
         {
             if (instantiateedGameObject != null)
             {
-                instantiateedGameObject.transform.position = location + tilemap.GetComponent<Tilemap>().tileAnchor;
+                Tilemap tmpMap = tilemap.GetComponent<Tilemap>();
+                instantiateedGameObject.transform.position = tmpMap.CellToWorld(location) + Vector3.Scale(tmpMap.tileAnchor, tmpMap.cellSize);
```

Comment:
I think this may give a better position depending on the Grid settings:
                instantiateedGameObject.transform.position = tmpMap.LocalToWorld(tmpMap.CellToLocalInterpolated(location + tmpMap.tileAnchor));

Reply:
Let me know if this works out for you (or if it doesn't)?

Reply:
Thanks for looking into this and fixing this!

Reply:
Thanks! It works for me. New version commited